README.md

```diff
@@ -2,33 +2,109 @@
 
 [![Build Status](https://travis-ci.com/PaNOSC-ViNYL/libpyvinyl.svg?branch=master)](https://travis-ci.com/PaNOSC-ViNYL/libpyvinyl)
```

Comment:
Build status icon is still taken from travis, should be gihub actions, right?

Reply:
Fixed: https://github.com/PaNOSC-ViNYL/libpyvinyl/tree/develop#readme